tests/vm_test.go

```diff
@@ -92,13 +93,13 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			template.Spec.Domain.Devices.Disks = append(template.Spec.Domain.Devices.Disks, v1.Disk{
 				Name: "testdisk",
 			})
-			newVMI := NewRandomVirtualMachine(template, false)
-			newVMI.TypeMeta = v12.TypeMeta{
+			newVM := NewRandomVirtualMachine(template, false)
+			newVM.TypeMeta = v12.TypeMeta{
 				APIVersion: v1.GroupVersion.String(),
 				Kind:       "VirtualMachine",
 			}
 
-			result := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body(newVMI).Do()
+			result := virtClient.RestClient().Post().Resource("virtualmachines").Namespace(tests.NamespaceTestDefault).Body(newVM).Do()
```

Comment:
again not really related... but there is a nicer function in virtClient for creating a VM :)

Reply:
Right, and it's used elsewhere in this module but note that this particular test checks the response of the POST call so I think it's ok here

Reply:
oh, I missed that! Then it's ok of course